### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by mimansha-swarup**
https://user-images.githubusercontent.com/69811810/159668625-389c1031-de20-46e8-85db-6c14225cc9fb.mp4

### CODE REVIEW

File: src/Helper/FilterFunction.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export const sortbyPrice = (listData, sortType) =>
```
Comment (poornima-alugubelly) [inline src/Helper/FilterFunction.js:1]: all utilities are pure nice 👍

File: src/Page/Product/ProductsPage.js [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+    filterState;
 
-        Products
-        <button className="btn" onClick={handleFilterOpen} >Filters</button>
-        {
-          isFilterOpen && <div className="absolute-sidebar">
-      <Filter  />
+  const sortedList = sortbyPrice(products, sortBy);
+  // const stockList = displayinStock(sortedList,showInventoryAll) Todo:later add out of stock
+  const priceList = filterByPrice(sortedList, maxPrice);
```
Comment (poornima-alugubelly) [inline src/Page/Product/ProductsPage.js:29]: remove console logs

**Review by poornima-alugubelly — APPROVED** [review]

File: src/Reducer/FilterReducer.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        showInventoryAll: true,
+        showFastDeliveryOnly: false,
+        maxPrice: 1000,
+        rating: 5,
+        searchQuery: "",
+        category: [],
+      };
+      
+      default:
+      console.log("clear deafaukt is working")
```
Comment (manojasarsa) [inline src/Reducer/FilterReducer.js]: You can remove these console.log

**Review by manojasarsa — APPROVED** [review]
Rest looks clean. 
Good to merge!

**Pull request merged**